scalastyle-config.xml

```diff
@@ -136,7 +136,7 @@ This file is divided into 3 sections:
   <check level="error" class="org.scalastyle.scalariform.NotImplementedErrorUsage" enabled="true"/>
 
   <check customId="funsuite" level="error" class="org.scalastyle.scalariform.TokenChecker" enabled="true">
-    <parameters><parameter name="regex">^FunSuite[A-Za-z]*$</parameter></parameters>
+    <parameters><parameter name="regex">^AnyFunSuite[A-Za-z]*$</parameter></parameters>
```

Comment:
so there will be no check for `FunSuite`?

Reply:
`FunSuite` is deprecated in scalatest 3.1, and replaced by `AnyFunSuite`

Reply:
I know, it's deprecated but not removed so we still need this checker?

Reply:
updated.